pom.xml

```diff
@@ -126,6 +126,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>credentials</artifactId>
+      <version>1378.v81ef4269d764</version>
```

Comment:
Why are we providing the version here?

Reply:
just need to force version containing this change https://github.com/jenkinsci/jenkins-test-harness/pull/829

Reply:
not sure I follow, if you want that change, then change the property `jenkins-test-harness.version`?

do you mean that you want a version containing https://github.com/jenkinsci/credentials-plugin/pull/558 instead?

Reply:
yup sounds like a typo